tests/unit/backend/wmg/fixtures/test_snapshot.py

```diff
@@ -172,7 +172,7 @@ def build_cell_orderings(cell_counts_cube_dir_, cell_ordering_generator_fn) -> D
 def create_dataset(dataset_id_ordinal: int) -> str:
     coll_id = f"dataset_id_{dataset_id_ordinal}_coll_id"
     with db_session_manager() as session:
-        if coll := Collection.get(session, (coll_id, CollectionVisibility.PUBLIC)):
+        if coll := Collection.get(session, coll_id):
```

Comment:
@atolopko-czi pls confirm this is an acceptable change; the tests pass, just want to make sure there isn't a hidden requirement to verify that a collection is public at this line

Reply:
This is good--dataset visibility is not a concern of any wmg API tests.